test/baselines/TS_Prettier/package.json

```diff
@@ -19,33 +19,33 @@
     "url": "https://github.com/AlCalzone/ioBroker.test-adapter"
   },
   "dependencies": {
-    "@iobroker/adapter-core": "^2.5.1"
+    "@iobroker/adapter-core": "^2.6.0"
   },
   "devDependencies": {
     "@iobroker/adapter-dev": "^1.0.0",
-    "@iobroker/testing": "^2.5.4",
-    "@types/chai": "^4.3.0",
+    "@iobroker/testing": "^3.0.1",
+    "@types/chai": "^4.3.1",
     "@types/chai-as-promised": "^7.1.5",
-    "@types/mocha": "^9.1.0",
-    "@types/node": "^14.18.12",
+    "@types/mocha": "^9.1.1",
+    "@types/node": "^14.18.16",
     "@types/proxyquire": "^1.3.28",
     "@types/sinon": "^10.0.11",
     "@types/sinon-chai": "^3.2.8",
-    "@typescript-eslint/eslint-plugin": "^5.12.0",
-    "@typescript-eslint/parser": "^5.12.0",
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
+    "@typescript-eslint/parser": "^5.23.0",
     "chai": "^4.3.6",
     "chai-as-promised": "^7.1.1",
-    "eslint": "^8.9.0",
-    "eslint-config-prettier": "^8.3.0",
+    "eslint": "^8.15.0",
+    "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "mocha": "^9.2.0",
-    "prettier": "^2.5.1",
+    "mocha": "^10.0.0",
+    "prettier": "^2.6.2",
     "proxyquire": "^2.1.3",
     "rimraf": "^3.0.2",
-    "sinon": "^13.0.1",
+    "sinon": "^14.0.0",
```

Comment:
sinon 14 has no node 12 support. should we pin it?

Reply:
Can you do this in #910 where the user can choose the node version?